templates/mongodb-atlas-main.template.yaml

```diff
@@ -88,27 +88,14 @@ Parameters:
     - "M20"
     - "M30"
     - "M40"
-    - "R40"
```

Comment:
i made an error, double checked with atlas go-client and indeed most of these are supported MongoDB Cluster Tiers for AWS. can you update to use this list instead: https://github.com/mongodb/go-client-mongodb-atlas/blob/master/mongodbatlas/clusters.go#L250

also please add "R700" + delete "M400" as well as "M700" as these two 2 are not supported.

Reply:
We have M400 and M700 on UI. Any idea which API UI is using?
<img width="971" alt="Screenshot 2023-02-01 at 12 11 05 PM" src="https://user-images.githubusercontent.com/7871501/215970336-05b2f825-53d2-4964-abbd-b4d2e53b1207.png">

Reply:
updated as per the suggestion:
https://github.com/aws-quickstart/quickstart-mongodb-atlas/pull/30/commits/f0813265c133ccf7c2a459159358541bc2e6b768